torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -526,7 +526,7 @@ struct CodeImpl {
       }
     } else {
       int reg = registerFor(input);
-      bool moved = input->uses().size() == ++use_count_[input];
+      bool moved = false; // input->uses().size() == ++use_count_[input];
```

Comment:
This is the temporary change I was referring to in my comment. I do not intend to land this, but the rest of my changes don't work without it for the reason listed in my top-level comment.